### PR TITLE
feat: DAZ-13931 add botbot and printing as a service um

### PR DESCRIPTION
**Pull request by Timberbain**

### CODE REVIEW

**Review by cahrsjo — APPROVED** [review]
🤖 🤖

**Review by edjmacedo — APPROVED** [review]
🔥 🔥 🔥

**Review by Babak-Gholamzadeh — APPROVED** [review]

**Review by carlaxel — APPROVED** [review]

**Comment by jasonlohyp** [comment]
Rock🔥 🔥 🔥

**Review by Babak-Gholamzadeh — DISMISSED** [review]

File: streams/botbot-ai-assist/bot bot ai assist - guide.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Step 5:
+Run stream "1. Provisioning Meter Types". After the stream has completed you will have a new meter type named "AI Assist" in the Usage metering section.
+
+Step 6: 
+Navigate to "Usage metering" and copy the ids of the "AI Assist" meter type for later.
+Also, open the stream configuration of "2. Botbot usage data billing stream" and copy the meter key from the Meter processor for later.
+
+Step 7:
+Open the stream configuration of "2. Provisioning Meters" and open the script named "Prepare Meters". Paste in the meter type ids and meter key that you got from the previous step.
```
Comment (Babak-Gholamzadeh) [inline streams/botbot-ai-assist/bot bot ai assist - guide.md:28]:
```suggestion
Open the stream configuration of "3. Provisioning Meters" and open the script named "Prepare Meters". Paste in the meter type ids and meter key that you got from the previous step.
```

File: streams/botbot-ai-assist/bot bot ai assist - guide.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Step 8:
+Run stream "2. Provisioning Meters". After the stream has completed, you will have 5 meters across your newly created meter types.
```
Comment (Babak-Gholamzadeh) [inline streams/botbot-ai-assist/bot bot ai assist - guide.md:31]:
```suggestion
Run stream "3. Provisioning Meters". After the stream has completed, you will have 5 meters across your newly created meter types.
```

File: streams/printing-as-a-service-um/printing-as-a-service-um.json
Comment (Babak-Gholamzadeh) [inline streams/printing-as-a-service-um/printing-as-a-service-um.json:1]: the other stream names start with number `1., 2., 4.` but not this one
Don't if we should add or not

**Review by Saeger — APPROVED** [review]
LGTM

**Review by Babak-Gholamzadeh — APPROVED** [review]

**Review by notrev — APPROVED** [review]
ok

**Review by Babak-Gholamzadeh — APPROVED** [review]

**Review by carlaxel — APPROVED** [review]

**Pull request merged**